au4/src/au3wrap/internal/au3project.cpp

```diff
@@ -87,6 +88,8 @@ bool Au3Project::load(const muse::io::path_t& filePath)
     TrackList& tracks = TrackList::Get(m_data->projectRef());
     for (auto pTrack : tracks) {
         pTrack->LinkConsistencyFix();
+        //! TODO AU4: adjust to actual project tempo
+        DoProjectTempoChange(*pTrack, 120);
```

Comment:
Could you please explain these changes?

Reply:
Project needs to have a tempo setup - if it doesn't when you try to copy&paste a clip then Paste() function throws an exception. So for now I've set a hardcoded 120 BPM (it's not used anywhere yet except copy&paste logic)